ImperatorToCK3/Source/CK3/Titles/LandedTitles.cpp

```diff
@@ -41,26 +43,32 @@ std::optional<std::string> CK3::LandedTitles::getCountyForProvince(const unsigne
 	for (const auto& [titleName, title] : foundTitles)
 	{
 		if (titleName.starts_with("c_") && !title->getCountyProvinces().empty())
-			if (title->getCountyProvinces().contains(provinceID)) return titleName;
+			if (title->getCountyProvinces().contains(provinceID))
```

Comment:
No need to nest ifs, you can put the conditional right above after additional &&. If the second one fails, third one won't execute and won't crash. In this particular case no need to poke if empty before poking with contains, compiler should be smart enough to return false if container is empty.

Reply:
Yeah contains() returns false if the container is empty, so the empty() is redundant.